packages/plugin-server-session/session.js

```diff
@@ -37,11 +37,6 @@ const sendSessionSummary = client => sessionCounts => {
     return
   }
 
-  if (!client.config.endpoints.sessions) {
```

Comment:
As above, commenting here too for an equivalent change… the config schema now validates that `endpoints.sessions` must now be defined so this branch of code is obsolete.